### ISSUE TITLE
Enable Spring module build/release in our engineering system

### ISSUE THREAD

**Issue by kurtzeborn** [Azure/azure-sdk#1488]

**Comment by mitchdenny** [comment]
@JimSuplizio and @kurtzeborn ... I feel like we are there with Spring. Looking at their pipelines it looks like they can build and release. They still might have some tidy up to do but I'm not seeing anything from an EngSys perspective that is blocking their releases that they won't be in a position to fix themselves - misplaced version tags etc.

**Comment by JimSuplizio** [comment]
Adding @weshaggard @scbedd 
I believe we're there as well. I'd [previously done the work to create empty javadoc and sources jars](https://github.com/Azure/azure-sdk-for-java/pull/11638) as per maven's instructions. At that time I verified the various releases using the azure-sdk-template library. Since that time, there was one new step added to the release steps which was the docs.ms step. On Tuesday (7/28/2020) I'd reached out to @scbedd and we tested the code in that step with the spring artifacts, the same set that is going to be released. We believe they should be fine.
